src/__tests__/Pages.test.tsx

```diff
@@ -146,6 +146,7 @@ const signupGroupValues: SignupGroupFormFields = {
       id: TEST_SIGNUP_ID,
       inWaitingList: false,
       lastName: 'Last name',
+      phoneNumber: '0441234567',
```

Comment:
This same change is introduced in another PR - perhaps no need to have it here, or introduce it only here in a separate commit?

Reply:
Removed it from this PR and created a separate commit in the another PR